### ISSUE TITLE
torch Specific installation version

### ISSUE THREAD

**Issue by liuhongyan123456** [xiaoyuan1996/AMFMN#8]
Hello blogger, I have the following problem while running the training code and would like to ask you the specific version information of torch installation
![1702285777964](https://github.com/xiaoyuan1996/AMFMN/assets/49739798/8e72054e-75d6-4b3b-bee8-7f27265a0471)

**Comment by tiantianbukuaile** [comment]
self.transform = transforms.Compose([
                transforms.Resize((278, 278)),
                transforms.RandomRotation((0, 90)),
                transforms.RandomCrop(256),
                transforms.ToTensor(),
                transforms.Normalize((0.485, 0.456, 0.406),
                                     (0.229, 0.224, 0.225))])
